Add a python module for ratbagd, replace ratbagctl with a python version

whot opened the pull request:
ratbagctl is a lot easier to understand and manipulate this way :)

whot committed:
Add a python module to talk to ratbagd

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>

dvdhrm commented:
Awesome! I should definitely spend more time writing python, this looks really nice.

whot committed:
Replace ratbagctl with a python version

It's a lot easier to interact with ratbagd from a python script than it is
with C code. Plus this means we test the bindings whenever we use ratbagctl.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>

whot commented:
sorry, I keep screwing up my branches these days. just force-pushed an update for 0528981. Only change is the removal of the ratbagctl.c file in the second commit now, it was in the first commit before.

bentiss commented:
Looks good to me
